indexer/packages/postgres/src/lib/helpers.ts

```diff
@@ -1,14 +1,15 @@
-// Gets `goodTilBlockTime` from an ISO string.
 import { DateTime } from 'luxon';
 
 import { IsoString } from '../types';
 import vaultAddresses from './vault-addresses.json';
 
+// Gets `goodTilBlockTime` from an ISO string.
 export function blockTimeFromIsoString(isoString: IsoString): number {
   const dateTime: DateTime = DateTime.fromISO(isoString, { zone: 'utc' });
   return Math.floor(dateTime.toMillis() / 1000);
 }
 
-export function getVaultAddresses(): string[] {
-  return vaultAddresses;
-}
+// Set of the addresses of vaults that quote on clob pairs 0 to 999.
```

Comment:
Could we add a TODO item to find a more permanent solution here, so we don't end up forgetting about this and run into issues with vault ID > 1000